src/main/resources/schema/mutation.graphql

```diff
@@ -0,0 +1,51 @@
+type Mutation {
+  "Authenticate a user using  username & password"
+  login(userName: String, password: String): User
+
+  "Create a new App Wish "
+  addWish(wishDetails: WishInput): Wish
+
+  "Edit an AppWish, providing the ID and the fields to update"
+  editWish(id: ID!, wishInput: WishInput): Wish
+
+  "Delete an AppWish by ID"
+  deleteWish(id: ID!): [Wish]
+
+  "Add a new comment"
+  addComment(userID: ID, msg: String): Comment
```

Comment:
Okay, so if I got you right, I need to bring in a targetID where the targetID can either be a Wish, a comment etc right?

Applying the changes as soon as possible

Reply:
TargetId would be an ID of some item e.g. a wish id.  TargetType, on the other hand, could indicate what's the type of targetId (is it a wish? a comment? something else?).

This way we could store all comments in one table. If we want all comments of a wish id=1, then we request all comments with targetId=1 and targetType=wish (+ their children comments).

I'm not sure if it's final (and solid) approach though - there is one task in progress to come up with the design of DB schema for storing comments. When it's done it will be probably much easier to design the graphql comments schemas.